cid/overlay/frameworks/base/core/res/res/values/config.xml

```diff
@@ -96,4 +93,5 @@
 
     <!-- Default value for led color when battery is fully charged: GREEN -->
     <integer name="config_notificationsBatteryFullARGB">0xFF00FF00</integer>
+    <add-resource type="integer" name="config_LTE_RSRP_threshold_type">0</add-resource>
```

Comment:
Is this resource actually useful to anyone? Just wondering if we'd be better off just deleting it instead of adding it.

Also, can you add a line before that, so we know if different from the LED config value. Thanks!

Reply:
Good catch !! It is unused. So have removed them.